waku/protocol/v2/waku_filter.nim

```diff
@@ -157,3 +189,12 @@ proc subscribe*(wf: WakuFilter, request: FilterRequest): Future[string] {.async,
     let conn = await wf.switch.dial(peer.peerId, peer.addrs, WakuFilterCodec)
     await conn.writeLP(FilterRPC(requestId: id, request: request).encode().buffer)
   result = id
+
+proc unsubscribe*(wf: WakuFilter, request: FilterRequest) {.async, gcsafe.} =
+  # @TODO: NO REAL REASON TO GENERATE REQUEST ID FOR UNSUBSCRIBE OTHER THAN CREATING SANE-LOOKING RPC.
```

Comment:
Maybe we can cut this request id stuff out into a separate GH issue?

Reply:
Opened #257 to keep track of outstanding work on `requestId`